### PR TITLE
[quick-list] cleanup quick list

### PR DESCRIPTION
**Pull request by adichioran**

### CODE REVIEW

File: src/dev/src/app/quick-list/quick-list.demo.html [hunk 1]
```diff
@@ -2,15 +2,18 @@ <h2>Quick List</h2>
 <div class="clr-example">
   <div class="main-container">
     <div class="clr-row">
```
Comment (Wallabeng) [inline src/dev/src/app/quick-list/quick-list.demo.html]: main-container and clr-row are destroying the layout in the demo. pls remove them and test if it looks the same (margins, paddings) as in the combobox demo

File: src/dev/src/app/quick-list/quick-list.demo.html [hunk 2]
```diff
@@ -2,15 +2,18 @@ <h2>Quick List</h2>
...
+          class="clr-form-control clr-col-12 clr-row"
+          clrControlClasses="clr-col-lg-3 clr-col-md-4 clr-col-sm-5"
+          [clrAllValues]="possibleActivityResults"
+          [(clrValues)]="selectedActivityResults"
+          [clrAddLabel]="getAddLabel()"
+          [clrBlankOption]="BLANK_OPTION"
+        >
+          <label class="clr-control-label  clr-col-md-2 clr-col-sm-3 required">Option list</label>
+        </clr-quick-list>
+      </form>
```
Comment (Wallabeng) [inline src/dev/src/app/quick-list/quick-list.demo.html]: just add a clr-input-container (like in combobox demo) to check if the layout inside a form is aligned with the standard elements

File: src/clr-addons/quick-list/add-option.html [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
-    <select class="quick-list-select" clrSelect name="options" [(ngModel)]="selectedValue">
-      <option *ngIf="blankOption.id === selectedValue" [selected]="true" [disabled]="true" [ngValue]="blankOption.id">
-        {{blankOption.label}}
-      </option>
-      <option *ngFor="let val of getValues()" [selected]="val.id === selectedValue" [ngValue]="val.id">
-        {{val.label}}
-      </option>
-    </select>
-  </clr-select-container>
+<div class="clr-select-wrapper quick-list-input">
```
Comment (Wallabeng) [inline src/clr-addons/quick-list/add-option.html]: is something wrong with using clr-select-container component and clrSelect directive?
if not please use them, so no adaption is needed if any css class is renamed

Comment (adichioran) [reply src/clr-addons/quick-list/add-option.html]: the select container is unusable for this component (content is no longer shown)

**Pull request merged**